seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/checkpoint/CheckpointCoordinator.java

```diff
@@ -170,7 +170,7 @@ public CheckpointCoordinator(
                 new ArrayDeque<>(coordinatorConfig.getStorage().getMaxRetainedCheckpoints() + 1);
         this.scheduler =
                 Executors.newScheduledThreadPool(
-                        2,
+                        1,
```

Comment:
Do we need to revert this?

Reply:
Oh I missed it. I think it should be `2`.